README.md

```diff
@@ -1,70 +1,71 @@
-# Getting Started with Create React App
+# BookStore--CMS
```

Comment:
- [ ] Please change the name of the app here, I think you are using the name of your previous project. Correcting it will make your README file more professional.